test/erf.jl

```diff
@@ -137,5 +137,9 @@
         @test erf(11, 10) ≈ -2.08848758232167861905709161e-45
         @test erf(-11, -10) ≈ 2.08848758232167861905709161e-45
         @test erf(-1, 1) ≈ 1.68540158589942973868244127017
+        @test erf(1e-30, 2e-30) ≈ 1.12837916709551257389615890312e-30
```

Comment:
The exactly rounded answer is `-1.1283791670955127e-30`

Reply:
I computed that with Mathematica in arbitrary precision, so I expect all the displayed digits to be exact (except maybe the last). But you have `7` on the last digit that differs from what I got in Mathematica? This is probably not important though.